tests/python/unittest/test_target_codegen_llvm.py

```diff
@@ -534,7 +534,7 @@ def _show_info():
             (2, 2),
             (-11, 11),
         ]:
-            if end < start or dend < dstart or (dend == 0 and dstart == 0):
```

Comment:
nit: could you please tally against https://github.com/apache/tvm/blob/902c2e2db70b75c36f9bd8c253707b1e8761cc18/tests/python/unittest/test_tir_nodes.py#L300 to check the divide by zero test case is covered there?

Reply:
This test seems to be checking for div by 0 at compile time due to the use of constants, in the case of `test_llvm_div`, the div by 0 occurs at runtime, so I don't think this is something we can check here